### PR TITLE
Improved UNet Scoring 0.8 on Test Set

### PR DESCRIPTION
**Pull request by default-jamc**
Hi Shakes & Tutors,

Please review this pull request of my implementation of the Improved UNet model. This implementation achieves > 0.8 on the test set, with > 67% of the test set attaining a DSC of 0.8 or above. The files which were added as part of this development are:

- `driver.py`: driver file for the model
- `model.py`: the improved UNet model, implemented in Tensorflow.
- `README.md`: a README file detailing the model architecture, loss function and results of training.
- `images/` a directory containing images used throughout `README.md`

If there are any improvements or changes I could make to this before this pull request is approved, please let me know! 

Jazmin (s4486659)

### CODE REVIEW

**Comment by wooboyeong** [comment]
General Comments
* Code OK for EASY
* Commit messsages OK
* Documentation OK
* Pull request OK

**Comment by shakes76** [comment]
## TF/Torch Usage
Adequate use and implementation
## Good Practice (Design/Commenting)
Good spacing and comments
No Header blocks -1
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage 
Module present 
No Data leakage 
Difficulty: Easy -10
## Commit Log
Meaningful commit messages
Progressive commits used
## Documentation
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) -1 
No feedback required
Request Description OK

**Comment by default-jamc** [comment]
Hi Shakes, 

May I ask why I lost 1 mark for "Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch)" ?

Thank you :)

**Comment by shakes76** [comment]
Error on my part, should be:
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch)
Mark amendment +1
